### ISSUE TITLE
Vue widget geosearch leaflet map

### ISSUE THREAD

**Issue by seifrached** [meilisearch/meilisearch-vue#115]
Hi 👋 

It’s possible to add geosearch widget with meilisearch instance using vue like this example ?

[https://codesandbox.io/embed/6c96p?file=/src/index.js&codemirror=1](https://codesandbox.io/embed/6c96p?file=/src/index.js&codemirror=1)

**Comment by bidoubiwa** [comment]
Hello! It should be working the [vue-instantsearch geo search widget](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/geo-search/vue/). Can you confirm?

**Comment by seifrached** [comment]
> Hello! It should be working the [vue-instantsearch geo search widget](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/geo-search/vue/). Can you confirm?


I try but the state always empty because

**Comment by bidoubiwa** [comment]
Mmh, not sure why. By following this as well: https://codesandbox.io/s/github/algolia/doc-code-samples/tree/master/vue-instantsearch/geo-search?file=/src/main.js:0-23 ?

**Comment by seifrached** [comment]
> Mmh, not sure why. By following this as well: https://codesandbox.io/s/github/algolia/doc-code-samples/tree/master/vue-instantsearch/geo-search?file=/src/main.js:0-23 ?

Yeah, I do the same but the the state 'items'  in this case is undefined !

**Comment by bidoubiwa** [comment]
Thanks a lot I will investigate this as soon as I can

**Comment by bidoubiwa** [comment]
Hey @seifrached I'm sorry I did not have time to look at this issue. I put it on my todo but I make it open for contribution during hacktorberfest
